NDecrypt/Program.cs

```diff
@@ -210,7 +210,8 @@ private static ITool DeriveTool(string filename, DecryptArgs decryptArgs)
         private static FileType DetermineFileType(string filename)
         {
             if (filename.EndsWith(".nds", StringComparison.OrdinalIgnoreCase)     // Standard carts
-                || filename.EndsWith(".srl", StringComparison.OrdinalIgnoreCase)) // Development carts/images
+                || filename.EndsWith(".srl", StringComparison.OrdinalIgnoreCase) // Development carts/images
+                || filename.EndsWith(".nds.enc", StringComparison.OrdinalIgnoreCase)) // gm9 output encrypted nds carts/images
```

Comment:
Please add alphabetically. Also, if you're doing this, mind's well add `.nds.dec` as well.

Reply:
- .enc is a relatively new output that was added with version 2.1.1 of godmode9 to dump nds carts with the secure area encrypted. While logical, .dec isn't a format you see in the wild nor is it an output for any dumping method I know of (as decrypted carts just use the .nds extension.) If you still want me to add it though, I'll go right ahead. 
 - Went ahead and alphabetized case/if statement and rewrote comment to be more verbose.